src/styles/aiReport.print.css

```
@@ -0,0 +1,295 @@
+/* AI 요약 PDF — iframe 전용 */
+:root {
+  --print-margin-x: 16mm;
+  --print-margin-top: 18mm;
+  --print-margin-bottom: 18mm;
+  --print-footer-h: 11mm;
+}
+
+@font-face {
+  font-family: "Pretendard";
```

Stylelint (17.12.0)

[error] Expected no quotes around "Pretendard" (font-family-name-quotes)

(font-family-name-quotes)